README.md

```diff
@@ -0,0 +1,143 @@
+![](header.png)
+
+[![Minio Version](https://img.shields.io/badge/Minio-latest-blue.svg)]() [![Dokku Version](https://img.shields.io/badge/Dokku-v0.11.2-blue.svg)]()
+
+# Run Minio on Dokku
+
+## Perquisites
+
+### What is Minio?
+
+Minio is an object storage server, and API compatible with Amazon S3 cloud
+storage service. Read more at the [minio.io](https://www.minio.io/) website.
+
+### What is Dokku?
+
+[Dokku](http://dokku.viewdocs.io/dokku/) is the smallest PaaS implementation
+you've ever seen - _Docker powered mini-Heroku_.
+
+### Requirements
+* A working [Dokku host](http://dokku.viewdocs.io/dokku/getting-started/installation/)
+
+# Setup
+
+**Note:** We are going to use the domain `minio.example.com` for demonstration
+purposes. Make sure to replace it to your domain name.
+
+## Create the app
+Log onto your Dokku Host to create the Minio app:
+
+```bash
+dokku apps:create minio
+```
+
+## Configuration
+
+### Setting access keys
+
+Minio uses two access keys (`ACCESS_KEY` and `SECRET_KEY`) for authentication
+and object management. The following commands sets a random strings for each
+access key.
+
+```bash
+dokku config:set minio MINIO_ACCESS_KEY=$(echo `openssl rand -base64 45` | tr -d \=+ | cut -c 20)
+dokku config:set minio MINIO_SECRET_KEY=$(echo `openssl rand -base64 45` | tr -d \=+ | cut -c 32)
```

Comment:
These should be `cut -c 1-20` and `cut -c 1-32` respectively otherwise the output will be only one key.